styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java

```diff
@@ -83,7 +83,16 @@ public void printEvents(List<EventInfo> eventInfos) {
 
   @Override
   public void printBackfill(Backfill backfill) {
-    System.out.println(backfill);
+    System.out.println(String.format("id: %s", backfill.id()));
```

Comment:
What do you think about skipping this "header" when we print plain info about a single backfill? So the output is just a list of executions with their status (easy to grep)